edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_course-experience.scss

```diff
@@ -1,555 +1,41 @@
-// ------------------------------
```

Comment:
Was it necessary to remove all these rules? @luismorenolopera

Reply:
Yes, the result of this page was similar to another existing theme (pearson-pols)